modules/claims_api/app/swagger/claims_api/claims/claims_response_swagger.rb

```diff
@@ -299,10 +299,12 @@ class ClaimsResponseSwagger
               key :description, 'Current status of the claim'
               key :enum, [
                 '"Claim received"',
+                '"Complete"',
                 '"Initial review"',
+                '"Errored"',
                 '"Evidence gathering, review, and decision"',
-                '"Preparation for notification"',
-                '"Complete"'
+                '"Pending"',
```

Comment:
these new values should *not* be capitalized. This collection of values is a combination of statuses we have in addition to values we get back from EVSS. Unfortunately we don't capitalize our statuses but EVSS does :(

Reply:
Ahhhhh, ok.
Updated [here](https://github.com/department-of-veterans-affairs/vets-api/pull/5904/commits/1bbdc8a7a65bed50126677fa3c13b27660964296).